@shared/api/types.ts

```diff
@@ -18,6 +18,8 @@ export interface UserInfo {
   publicKey: string;
 }
 
+export type MigratableAccount = Account & { keyIdIndex: number };
```

Comment:
when we migrate an account, we need to know where the in the keyIdList this specific account falls so we can seamlessly replace the old account with the new one